unstructured/metrics/utils.py

```diff
@@ -98,23 +98,25 @@ def _display(df):
 
 
 def _write_to_file(
-    dir: str, filename: str, df: pd.DataFrame, mode: str = "w", overwrite: bool = True
+    directory: str, filename: str, df: pd.DataFrame, mode: str = "w", overwrite: bool = True
 ):
     """
     Save the metrics report to tsv file. The function allows an option 1) to choose `mode`
     as `w` (write) or `a` (append) and 2) to `overwrite` the file if filename existed or not.
     """
     if mode not in ["w", "a"]:
         raise ValueError("Mode not supported. Mode must be one of [w, a].")
-    if dir and not os.path.exists(dir):
-        os.makedirs(dir)
+    if directory and not os.path.exists(directory):
+        os.makedirs(directory)
```

Comment:
May want to use `os.makedirs` with `exist_ok=True`, or even refactor to `pathlib.Path`
```suggestion
    if directory:
        Path(directory).mkdir(exist_ok=True)
```